tests/integration_tests/test_binary_feature.py

```diff
@@ -0,0 +1,46 @@
+import pytest
```

Comment:
Thanks for adding this test!

Could you move this to `tests/ludwig/features`?

Reply:
moved, although Jim's tests that follow this pattern live in the integration tests directory

Reply:
I see. I can add a fix for that during the testing phase.